english/swami-vivekananda/teachings-of-swami-vivekananda/quotes.md

```diff
@@ -24,20 +24,55 @@ Try to be moral, try to be brave, try to be sympathising.
 
 
 3. Have faith that you are all, my brave lads, born to do great things! 
-

```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.